internal/pkg/metrics/incoming/ceilometer.go

```diff
@@ -201,22 +201,22 @@ func (c *CeilometerMetric) GetLabels() map[string]string {
 		labels[c.Plugin] = c.PluginInstance
 	}
 	labels["publisher"] = c.Publisher
-	if ctype, ok := c.Payload["counter_type"]; ok {
-		labels["type"] = ctype.(string)
+	if ctype, ok := c.Payload["counter_type"].(string); ok {
```

Comment:
This will panic now if counter_type key will be missing completely. Are we sure this situation will not happen? I would rather not count on it :). Please do this as inner conditional instead. I know it sucks, but that's how it works in go.

Reply:
One of the unit tests I made tries this with the fields completely missing and there is no panic, it works as expected.